addons/editor-dark-mode/addon.json

```diff
@@ -1511,6 +1511,31 @@
         "border": "#00000026"
       }
     },
+    {
+      "name": "Scratch Lab",
+      "id": "lab",
+      "description": "The colors used by Scratch Lab",
+      "values": {
+        "page": "#e5f0ff",
+        "primary": "#855cd6",
```

Comment:
The purples should be blue if the goal is to look how https://lab.scratch.mit.edu does now.